src/textures/VideoTexture.js

```diff
@@ -1,15 +1,10 @@
 import { RGBFormat, LinearFilter } from '../constants.js';
 import { Texture } from './Texture.js';
 
-function VideoTexture( video, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy ) {
+function VideoTexture( video, mapping, wrapS, wrapT, magFilter = LinearFilter, minFilter = LinearFilter, format = RGBFormat, type, anisotropy ) {
```

Comment:
Please avoid patters like this, see https://github.com/mrdoob/three.js/pull/20586#discussion_r514966578.